### ISSUE TITLE
Simplify the creation of new apps

### ISSUE THREAD

**Issue by jtanguy** [CleverCloud/clever-tools#51]
It would be nice if the creation of a new app could be as simple as linking an existing app.

```diff
- clever create -t node -o orga_<uuid> app-name
+ clever create -t node orga-name/app-name
```

**Comment by divarvel** [comment]
I'm taking the opportunity to improve the behaviour of `clever link` as well, especially in the case where the application name contains a '/'

## First possibility

```
# default behaviour, refers to an app inside the personal organisation
clever (link|create) app_name
# default behaviour, name is parsed as org_name / app_name
clever (link|create) org/app
# if a orgId is explicitely specified, then the name is used as is
clever (link|create) -o orga_<uuid> app/name
# if a orgId is explicitely specified with the value 'self', then the name is used as is
clever (link|create) -o self app/name
```


## Second possibility

Allow the `-o` flag to have the organisation name instead of an id, and never parse the name.
```
clever (link|create) app_name
clever (link|create) -o (org_id|org_name) app_name
```
That way it's less ambiguous, (no need to parse the given name).

Thoughts? @jtanguy

**Comment by jtanguy** [comment]
I'm not sure about an `-o` flag, since it's usually used for specifying an output file. Maybe a `-u` for user, since organizations can be seen as users?
But the `-o` flag is already there, so maybe not change that.

I would favor the parsing of `orga/app` name, or requiring that these commands take two arguments:
```
clever (link|create) org_name app_name
```
(And then, you must make sure that there is no org named _self_ or _me_, or whatever name you choose for one's own account)

**Comment by divarvel** [comment]
The issue is that `org_name` would be an optional argument, but not in the last position.

**Comment by divarvel** [comment]
Fixed by #67
